FLGradientInversion/README.md

```diff
@@ -74,8 +77,14 @@ BibTeX
 ```
 @article{hatamizadeh2022gradient,
   title={Do Gradient Inversion Attacks Make Federated Learning Unsafe?},
-  author={Hatamizadeh, Ali and Yin, Hongxu and Molchanov, Pavlo and Myronenko, Andriy and Li, Wenqi and Dogra, Prerna and Feng, Andrew and Flores, Mona G and Kautz, Jan and Xu, Daguang and Roth, Holger R},
+  author={Hatamizadeh, Ali and Yin, Hongxu and Molchanov, Pavlo and Myronenko, Andriy and Li, Wenqi and Dogra, Prerna and Feng, Andrew and Flores, Mona G and Kautz, Jan and Xu, Daguang and others},
```

Comment:
This reverted https://github.com/NVlabs/DeepInversion/pull/24 by mistake?

Reply:
Switched back to Google Scholar BibTeX [recommendation](https://scholar.googleusercontent.com/scholar.bib?q=info:UjYByNJX9YsJ:scholar.google.com/&output=citation&scisdr=CgVZmh0VEKuRkUgQcvw:AAGBfm0AAAAAY8YWavyBhNRuTq5rKOc28voz7Omuvs19&scisig=AAGBfm0AAAAAY8YWahcobhJYVr-gBy_fcpW8ebS9YZ19&scisf=4&ct=citation&cd=-1&hl=en). We will revise soon with the IEEE early access format, once it is out.